src/sx127x_linux_spi.c

```diff
@@ -45,24 +46,22 @@ int sx127x_spi_read_buffer(int reg, uint8_t *buffer, size_t buffer_length, void
   if (buffer_length < 1) {
     return 0;
   }
-  uint8_t *tmp = malloc(buffer_length + 1);
```

Comment:
> Why did you change dynamic allocation to static?

Sorry, stack allocation is a more appropriate term instead of "static allocation". This to spare some ``malloc/free`` calls with all the drawbacks them com with (speed, memory fragemntation, ...).

> Can you update sx127x_spi_write_buffer to make consistent?

Yes, will do :+1: